src/main/java/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -283,7 +284,9 @@ public final void createDatabase(DatabaseType dbType, Connection conn) throws SQ
                     LOG.debug(String.format("Starting %d %s.LoaderThreads [maxConcurrent=%d]", loaderThreads.size(), loader.getClass().getSimpleName(), maxConcurrent));
                 }
 
-                ThreadUtil.runNewPool(loaderThreads, maxConcurrent);
+                if (!loaderThreads.isEmpty()) {
```

Comment:
this correct #67, but can be removed if #68 is merged

Reply:
#68 has been merged.